autoPyTorch/ensemble/ensemble_builder.py

```diff
@@ -1033,20 +1027,19 @@ def get_n_best_preds(self) -> List[str]:
         num_keys = len(sorted_keys)
         # remove all that are at most as good as random
         # note: dummy model must have run_id=1 (there is no run_id=0)
-        dummy_scores = list(filter(lambda x: x[2] == 1, sorted_keys))
-
+        dummy_losses = list(filter(lambda x: x[2] == 1, sorted_keys))
         # Leave this here for when we enable dummy classifier/scorer
-        if dummy_scores:
+        if dummy_losses:
```

Comment:
Could we explicitly test if the list has dummy loss, i.e, if len(dummy_losses) > 0 or something. I think it will make the code more readable.

Reply:
Thanks,  I just pushed a new commit with this.

Reply:
Thanks,  I just pushed a new commit with this.